app/src/main/java/org/jellyfin/androidtv/data/eventhandling/TvApiEventListener.java

```diff
@@ -193,7 +193,7 @@ public void onResponse(ItemsResult response) {
                                     TvApp.getApplication().getCurrentActivity().startActivity(intent);
                                     break;
                                 case "Audio":
-                                    mediaManager.playNow(Arrays.asList(response.getItems()), startIndex);
+                                    mediaManager.playNow(Arrays.asList(response.getItems()), startIndex, false);
```

Comment:
For these values could we use a named bool so future devs don't have to dive into the docs?

```kotlin
shuffle = false
mediaManager.playNow(Arrays.asList(response.getItems()), startIndex, shuffle);
```

I find it prevents code from gradually turning into this:

```kotlin
mediaManager.foo(response, true, false);
```

Reply:
I'm definitely open to this.
I do think this accommodation is only useful in IDEs without syntax highlighting / introspection features.
In android studio, parameters like this have their name shown beside them.

Reply:
Ah feel free to disregard this 👍 
I tend to use both vim and the IDE so this is why I spotted it intislly.
But I'm also not against either jumping to the source or using the proper tool compared to adding noise to the source code. Plus multiple booleans is indicative of a hidden dataclass we can pull out

Reply:
I'm a fan of vim too
Have you tried plugins like this:
https://github.com/uiiaoo/java-syntax.vim

I just tested with neovim and it does add nice colors to the code but it still doesn't provide the parameter names for methods.